File: 02.PYTHON/pg-166-169/pg_166_01.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("READ AN INTEGER AND DETERMINE IF IT'S LAST NUMBER IS 4");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("");

num = int(input("Write an integer: "));
four = 4;

if(num < 0):
    num *= -1;

if(num % 10 == 4):
    print("The last digit of your number is ", four)
else:
    print("The last number of your number ISN'T ", four)

File: 01.JAVA/pg-166-169/pg_166_01.java
import java.util.Scanner;

public class pg_166_01 
{
    public static void main(String[] args)
    {
        Scanner enter = new Scanner(System.in);
        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ AN INTEGER AND DETERMINE IF IT'S LAST NUMBER IS FOUR");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");

        System.out.print("Write an integer: ");
        int num = enter.nextInt();
        int four = 4;

        if(num < 0)
        {
            num *= -1;
        }
        
        if(num % 10 == 4)
        {
            System.out.println("The last digit of your number is " + four);
        }
            else
                {
                    System.out.println("The last digit of your number ISN'T " + four);
                }
    }   
}
